contracts/OrderLib.sol

```diff
@@ -152,17 +152,20 @@ import "./helpers/AmountCalculator.sol";
     }
 
     /**
-      * @dev Validates the extension associated with an order. Reverts if invalid.
+      * @dev Validates the extension associated with an order.
       * @param order The order to validate against.
       * @param extension The extension associated with the order.
+      * @return True if the extension is valid, false otherwise.
+      * @return The error selector if the extension is invalid, 0x00000000 otherwise.
```

Comment:
Natspec generator treats the first word after return keyword as return variable name.
```suggestion
      * @return success True if the extension is valid, false otherwise.
      * @return errorSelector The error selector if the extension is invalid, 0x00000000 otherwise.
```